### PR TITLE
add Fedora EOL, update other releases

### PR DESCRIPTION
**Pull request by konstruktoid**
Signed-off-by: Thomas Sjögren <konstruktoid@users.noreply.github.com>
ref #944 

Update FreeBSD, OpenBSD, Ubuntu and Fedora end-of-life.

### CODE REVIEW

File: db/software-eol.db [hunk 1]
```diff
@@ -102,8 +104,17 @@ os:NetBSD 9.0::-1:
...
-#
-# Ubuntu - https://wiki.ubuntu.com/Kernel/LTSEnablementStack
+os:OpenBSD 6.0:2017-09-10:0:
+os:OpenBSD 6.1:2018-04-15:0:
+os:OpenBSD 6.2:2018-10-18:0:
+os:OpenBSD 6.3:2019-05-03:0:
+os:OpenBSD 6.4:2019-10-17:0:
+os:OpenBSD 6.5:2020-05-19:0:
+os:OpenBSD 6.6:2020-10-01:0:
+os:OpenBSD 6.7:2021-05-01:0:
```
Comment (Jaimie85) [inline db/software-eol.db:114]: Adding converted dates:
```suggestion
os:OpenBSD 6.0:2017-09-10:1505001600:
os:OpenBSD 6.1:2018-04-15:1523750400:
os:OpenBSD 6.2:2018-10-18:1539820800:
os:OpenBSD 6.3:2019-05-03:1556841600:
os:OpenBSD 6.4:2019-10-17:1571270400:
os:OpenBSD 6.5:2020-05-19:1589846400:
os:OpenBSD 6.6:2020-10-01:1601510400:
os:OpenBSD 6.7:2021-05-01:1619827200:
```

**Comment by konstruktoid** [comment]
Also cherry-picked https://github.com/CISOfy/lynis/pull/945/commits/78e7ce36afbbcba56fd65723f4d455de4d19ed6e from #948

**Comment by larsnaesbye** [comment]
Cool. Let's get this merged :-)

**Comment by mboelen** [comment]
Merged, thanks!

**Pull request merged**